### ISSUE TITLE
Two questions: 1) filter first or merge first, 2) B allele freq figure for duplication CNVs

### ISSUE THREAD

**Issue by xiaoqing-liu** [WGLab/PennCNV#106]
Dear Dr. Wang:
I have two questions after applying PennCNV to our twin data:

At the ‘annotation.md’ page (https://github.com/WGLab/PennCNV/blob/master/docs/user-guide/annotation.md), it listed the following two steps:
Filtering CNV calls by user-specified criteria
Merging adjacent CNV calls

My question is if we could conduct the merging first before filtering.

I ask this because we found that for one of our monozygotic twin pairs, one twin had a CNV at a locus but the other twin didn’t have this CNV. However, after we conducted merging first before filtering, the other twin also had a CNV at this locus.

My second question is about the figures. For one of the twins, PennCNV detected one duplication CNV at a locus. Is it correct that for duplications, the B allele frequency figure should have values at 0, 0.33, 0.67, and 1? We didn’t see this for this duplication CNV, instead we have the values 0 and 1 mostly but also some 0.5 at this region. Could it still be a valid duplication CNV without the 0.33 and 0.67 values for B allele freq?

Thank you in advance for your help with my questions!
Xiao-Qing

**Comment by kaichop** [comment]
Yes, it is best to merge before filtering. So two smaller CNVs are merged
into the real longer one, and then it will not be removed due to being
toosmall.
If you show the figure, I can make a guess. It all depends on the actual
length of region and the heterozygosity rate of that broader region (it
sounds like a LOH region)

**Comment by xiaoqing-liu** [comment]
Thank you so much for your reply, Dr. Wang!

Here I attached the figures. 1) a CNV (in red and is shorter) from filtering first, 2) a CNV (much longer) from merging first.

Thanks again!
Xiao-Qing


Xiao-Qing (Michelle) Liu 刘晓青

Principal Investigator and Senior Epidemiologist
Department of Obstetrics, Gynecology, and Reproductive Sciences

Department of Biochemistry and Medical Genetics

Faculty of Health Sciences, University of Manitoba

Co-Director, The Statistical Genetics and Bioinformatics Core Platform, Rady Faculty of Health Sciences
Scientist, Children's Hospital Research Institute of Manitoba

John Buhler Research Centre
807J - 715 McDermot Avenue
Winnipeg, Manitoba
Canada R3E 3P4
(204) 887-6986
***@***.***



[cid:3251587d-626b-42b5-8df4-45a18e108c80]

[cid:f352b6ee-f2d1-445f-a2d6-261279966719]

________________________________
